tests/spread/integration/adduser/task.yaml

```diff
@@ -0,0 +1,13 @@
+summary: Integration tests for adduser 
+
+execute: |
+  rootfs="$(install-slices adduser_bins)"
+  mkdir -p "$rootfs/{bin,lib,lib64}"
+  cp /bin/bash "$rootfs/bin"
```

Comment:
This also seem not so clean, you are not certain you get everything bash needs to run, why not just include `bash_bins` slice when you chisel?